.github/workflows/ci.yml

```diff
@@ -402,6 +402,38 @@ jobs:
           nistkat: true
           kat: false
           acvp: false
+  # ensure that kem.h and mlkem_native.h; api.h and native backends are compatible
+  check-apis:
+    needs: [quickcheck, quickcheck-windows, quickcheck-c90, quickcheck-lib, examples, lint, lint-markdown-link]
```

Comment:
This can be part of the basic CI tests, I think, so the `needs` line can go.

Reply:
fixed.